server/src/main/java/com/objectcomputing/member/MemberProfile.java

```diff
@@ -0,0 +1,68 @@
+package com.objectcomputing.member;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.sql.Date;
+
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@Table(name ="member_profile")
+public class MemberProfile {
+
+    public MemberProfile() {}
+
+    public MemberProfile(String name,String role,Long pdlId,String location
+                        ,String workEmail, String insperityId, Date startDate,
+                        String bioText) {
+                        this.name=name;
+                        this.role=role;
+                        this.pdlId=pdlId;
+                        this.location=location;
+                        this.workEmail=workEmail;
+                        this.insperityId=insperityId;
+                        this.startDate=startDate;
+                        this.bioText=bioText;
+                        }
```

Comment:
You have `@NoArgsConstructor`, but you have implemented that constructor...so it's not going to do anything.  If you want to use Lombok, I would kill line 24-37 and add `@AllArgsConstructor` to the class.

Reply:
So i didn't remove it because lombok is actually not working. I was playing with dependencies and still can't make it to work so thought of checking the constructor in case i can't find a solution.

Reply:
In that case, I'd kill lombok completely and add your getters and setters.  We can try to revive lombok later.

Reply:
@mkimberlin If you are ok then can we merge this PR ? Pramukh can then pull these changes and do a PR to remove lombok and add controller logic.